content/en/agent/troubleshooting/ntp.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Network Time Protocol (NTP) issues
+kind: documentation
+disable_toc: true
+aliases:
+    - /agent/faq/network-time-protocol-ntp-offset-issues
+---
+
+If you have noticed any of the following issues, they may be related to the NTP offset on the hosts that are reporting metrics through the Agent:
+
+* Incorrect alert triggers
+* Metric delays
+* Gaps in graphs of metrics
+
+To check the NTP offset for a host, run the Agent [status command][1], using the instructions appropriate for your OS, and look for the Clocks section:
+
+```
+  Clocks
+  ======
+    NTP offset: -0.0036 s
+    System UTC time: 2015-02-12 22:10:49.524660
+```
+
+Any significant offset can have undesired effects.To prevent NTP issues, leverage Datadog's monitor for NTP offset to alert you when there is drift on a host thanks to the [NTP Integration][2].
```

Comment:
```suggestion
Any significant offset can have undesired effects. To prevent NTP issues, leverage Datadog's monitor for NTP offset to alert you when there is drift on a host thanks to the [NTP Integration][2].
```